lib/plugins/aws/package/compile/events/stream.js

```diff
@@ -233,6 +237,10 @@ class AwsCompileStreamEvents {
                 event.stream.bisectBatchOnFunctionError;
             }
 
+            if (event.stream.functionResponseType != null) {
+              streamResource.Properties.FunctionResponseTypes = event.stream.functionResponseType;
```

Comment:
After reconfiguring schema to expect a string, we will have to wrap it to be an array here I believe